src/session/SessionMiddleware.ts

```diff
@@ -58,17 +58,15 @@ const sessionRequestHandler = (config: CookieConfig, sessionStore: SessionStore,
 
```

Comment:
Does a call come into this session request hander each and every time that an application configured to use the SessionMiddleware receives a HTTP request?

Reply:
In theory yes, the SessionMiddleware is set as middleware in each service, meaning every request runs though it before routing is done. This code lives in an "onHeaders" listener, which will execute the code when the service writes the headers to the response, which is that last thing done before the response is sent. I think this it to make sure this code is executed after the respective service has finished doing it's bit.

Reply:
👍 thanks.